ArcadiaParties.API/CustomMiddlewares/DatabaseAuthorizationMiddleware.cs

```diff
@@ -25,10 +25,10 @@ public async Task InvokeAsync(HttpContext context, IMediator mediator)
                 return;
             }
 
-            var getCurrentUserQuery = new GetCurrentUserQuery(user);
-            var currentUserFromDB = await mediator.Send(getCurrentUserQuery);
+            var getCurrentUserQuery = new GetUserRolesQuery(user);
```

Comment:
You should check what your code look like in whole after renaming.
var get**CurrentUser**Query = new Get**UserRoles**Query(user);

Reply:
Renamed